.github/workflows/annotate.yml

```diff
@@ -0,0 +1,55 @@
+name: Annotate checks
+
+on:
+  workflow_run:
+    workflows: ["ci"]
+    types:
+      - completed
+
+jobs:
+  annotate:
+    runs-on: ubuntu-latest
+    if: >
+      ${{ github.event.workflow_run.event == 'pull_request' }}
+    steps:
+      - name: 'Download artifact'
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const fs = require('fs');
+            const opts = github.rest.actions.listWorkflowRunArtifacts.endpoint.merge({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
```

Comment:
Add equivalent of https://github.com/trinodb/trino/blob/ba12095f2761cd2e7458a8be23d9b3489a5b0ca5/.github/workflows/milestone.yml#L9

Reply:
Done!